extensions/nanoarrow_ipc/src/nanoarrow/nanoarrow_ipc_decoder.c

```diff
@@ -930,7 +930,8 @@ static inline void ArrowIpcDecoderResetHeaderInfo(struct ArrowIpcDecoder* decode
 }
 
 // Returns NANOARROW_OK if data is large enough to read the message header,
```

Comment:
I guess this docstring is slightly misleading then because it may return 0 even if the buffer technically does not contain the entire header; it just has to contain the continuation token and size?